src/NzbDrone.Core/MetadataSource/SkyHook/Resource/TMDBResources.cs

```diff
@@ -34,6 +34,7 @@ public class MovieResult
         public int vote_count { get; set; }
         public bool video { get; set; }
         public float vote_average { get; set; }
+        public string Slug { get; set; }
```

Comment:
That does not work here. This class get's filed with the json info. The json info does not contain a Slug thing.